internal/user/service_test.go

```diff
@@ -665,3 +670,81 @@ func TestService_CreateTokenUpdateKeyErr(t *testing.T) {
 		}
 	})
 }
+
+func BenchmarkService_Login(b *testing.B) {
+	var (
+		ctx  = context.Background()
+		uDTO = &models.UserDTO{
+			Username: "bill",
+			Password: "password",
+		}
+	)
+
+	var cfg *config.Config
+
+	configPath := os.Getenv("CONFIG")
+	var once sync.Once
+	once.Do(func() {
+		cfg = config.GetConfig(configPath, &config.Config{})
+	})
```

Comment:
Why do we need once here?

Reply:
> 

For this case it's not important to use once. Just a copy-paste fail